### DISCUSSION TITLE
Create python bindings for the backend

### DISCUSSION THREAD

**Discussion by multun** [OpenRailAssociation/osrd#175] [Ideas]
We need a way to run interact with simulations from a python interface, either using FFI, or a networking api.

This interface will need to:

 - be able to manually run the simulation event loop, so we can stop at critical times
 - be able to change the ordering of trains at junctions
 - inject delays (we have to decide whether to pick a single way to do it or implement multiple failure modes)
 - observe train delays at a given time

**Comment by flomonster** [comment]
We need to make train disappear once they achieved their path. It could be an option given in the simulation payload.
